web/rest/brand/features/provider/matchList/putMatchList.feature

```diff
@@ -24,3 +24,25 @@ Feature: Update match lists
           "id": 3
       }
     """
+
+  Scenario: Match list updates ignores company values
+    Given I add Brand Authorization header
+    When I add "Content-Type" header equal to "application/json"
```

Comment:
Align this please

Reply:
I have formatted all matchList api tests using [this script](https://pypi.org/project/reformat-gherkin/). Is the only one I've found that is able to align format to the right. 

I'll add it to the pipeline in a later pull request, with all files updated.